### ISSUE TITLE
VoiceOver is unable to access heading in smart feeds

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [Ranchero-Software/NetNewsWire#2244]
Environment:
iOS 13.5.1
NNW 5.0.2

Steps:
1. Turn on VoiceOver
2. Navigate VoiceOver focus to a smart feed, like Today and double tap to open it
3. Touch the Feeds back button to move VoiceOver focus to it
4. Swipe right

Expected result:
VoiceOver should be able to navigate to the heading at the center of the navigation bar

Actual result:
VoiceOver skips this element.

**Comment by MarcoZehe** [comment]
Some more info: On iPhone, the heading is indeed missing. On iPad, it is there, but not identified as a heading (the heading trait is missing from the AccessibilityElement). On both iPhone and iPad, regular feeds both have their headings shown as plain text, but not identified as a heading.

**Comment by rizwankce** [comment]
@PeterSaathoff-Harshfield Im trying to make this fix for our upcoming v5.0.3 release. Im not quite getting your expected result. 

So when user comes back to "Feed" Screen, voice over skips the "Feeds" title. Is that what you mean here? I just tried it with my iPhone - voice over always goes for whatever I selected at last. For example when I select "Today" and go to timeline and come back to Feeds screen, voice over focusing on "Today".

Thanks in advance

**Comment by PeterSaathoff-Harshfield** [comment]
No, don't worry about where focus lands when navigating to a screen. That's mostly up to VoiceOver and you shouldn't interfere with it.

The issue is when you are on the Today smart feed, for example, and you place your finger on the "Today" heading at the top, VoiceOver will focus either "Feeds" (back button) or "Filter read articles" instead of being able to focus the heading.

**Comment by PeterSaathoff-Harshfield** [comment]
In other words the heading is not being correctly exposed as an accessible element.

**Comment by rizwankce** [comment]
Made a fix. Also added a unread count to support voice over.
